UnboundLib/Unbound.cs

```diff
@@ -351,46 +353,60 @@ internal static void SpawnMap(string message)
             try
             {
                 var currentLevels = MapManager.instance.levels;
-                var num = -1;
-                var num2 = 0f;
+                var levelId = -1;
+                var bestMatches = 0f;
+
+                // parse message
+                var formattedMessage = message.ToUpper()
+                    .Replace(" ", "_")
+                    .Replace("/", "");
+
                 for (var i = 0; i < currentLevels.Length; i++)
                 {
-                    var text = currentLevels[i].ToUpper();
-                    text = text.Replace(" ", "");
-                    text = text.Replace("ASSETS", "");
-                    text = text.Replace(".UNITY", "");
+                    var text = currentLevels[i].ToUpper()
+                        .Replace(" ", "")
+                        .Replace("ASSETS", "")
+                        .Replace(".UNITY", "")
+                        .Replace("/", "");
                     text = Regex.Replace(text, "/.*/", string.Empty);
-                    text = text.Replace("/", "");
-                    var text2 = message.ToUpper();
-                    text2 = text2.Replace(" ", "_");
-                    text2 = text2.Replace("/", "");
-                    var num3 = 0f;
-                    for (int j = 0; j < text2.Length; j++)
+
```

Comment:
`currentLevels[i].ToUpper().Replace("/", "")` needs to be executed after `text = Regex.Replace(text, "/.*/", string.Empty)` or else scene files in other folders than the main asset folder will not work with the command